src/modules/MouseWithoutBorders/App/Class/Common.Helper.cs

```diff
@@ -315,7 +313,7 @@ internal static void RunDDHelper(bool cleanUp = false)
                 HasSwitchedMachineSinceLastCopy = true;
 
                 // Common.CreateLowIntegrityProcess("\"" + Path.GetDirectoryName(Application.ExecutablePath) + "\\MouseWithoutBordersHelper.exe\"", string.Empty, 0, false, 0);
-                if (processes != null && processes.Length != 0)
+                if (Process.GetProcessesByName(HelperProcessName).Length == 0)
```

Comment:
The original code null checked the return by "?". Why are we removing that? If there's a possibility that there's a null return, we'll crash with the current code.